Kinvey/Kinvey/ValidationStrategy.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  DataValidationStrategy.swift
+//  Kinvey
+//
+//  Created by Victor Hugo Carvalho Barros on 2017-10-12.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import Foundation
+
+public enum ValidationStrategy {
+    
+    /// Percentagem between 0.0 and 1.0
+    case randomItems(percentage: Double)
```

Comment:
Can we name this `randomSample`? Usually "sampling" is the term used for what we're doing.